cluster/gce/gci/configure.sh

```diff
@@ -142,7 +142,7 @@ function download-or-bust {
   local -r hash="$1"
   shift 1
 
-  local -r urls=( $* )
+  local -r urls=( "$*" )
```

Comment:
pretty sure we are intentionally splitting here?

Reply:
oh, it's true.  I'll fix it. thank you.